tasks/prometheus.yml

```diff
@@ -41,5 +41,6 @@
     content: >
       {"service": {"id": "proxysql-native-exporter", "tags": ["prometheus-scraper", "proxysql-native"], "name": "prometheus-node-exporter", "port": 6070, "meta": {"role": "proxysql-native"}}}
   notify: consul reload
+  when: proxysql_version == '2.2.2'
```

Comment:
According to the ProxySQL releases the native Prometheus exporter was implemented since 2.1.0[https://github.com/sysown/proxysql/releases/tag/v2.1.0] so I would only demand those variables and tasks for those version starting from 2.1.0.